.luacheckrc

```diff
@@ -27,4 +27,5 @@ include_files = {
 exclude_files = {
     "lib/tarantool-python",
     "test/test-tarantool/*.test.lua",
+    "test/negative_tests/*.test.lua",
```

Comment:
The 'negative_tests' directory now on the top level, right? And it is named 'negative_test' (without 's').